toolkit/geoview-compose/src/main/java/com/arcgismaps/toolkit/geoviewcompose/MapView.kt

```diff
@@ -198,9 +200,16 @@ public fun MapView(
             })
 
         val mapViewScope = remember(mapView) { MapViewScope(mapView) }
```

Comment:
As with SceneView, the key shouldn't be necessary:
```suggestion
        val mapViewScope = remember { MapViewScope(mapView) }
```